pkg/core/bundle_pack.go

```diff
@@ -158,16 +166,25 @@ func uploadBundleFiles(
 				error: err,
 				file:  file,
 			}
+			break
```

Comment:
continuing through the loop in case of an error when `--skip-on-error` is not set causes a bug in error reporting.

although fixing this is a conceptually separate idea from the original intent this request, i've included it here because, overall, it does connect to debugging `bundle upload`.

Reply:
bundle.skipFile is to skip internal files, not sure how this is related to --skip-on-error